web/src/containers/ScheduleSummary.js

```diff
@@ -1,74 +1,57 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
-import ReactTable from 'react-table';
 
 import { t } from '../i18n';
 import Waypoint from './ConnectedWaypoint';
 import { Section, Subsection } from '../components/Section';
 import { selectActivitySchedule } from '../reducers/activities.selectors';
 
-const ScheduleSummary = ({ tableData }) => (
+const ScheduleSummary = ({ activities }) => (
   <Waypoint id="schedule-summary">
     <Section isNumbered id="schedule-summary" resource="scheduleSummary">
       <Subsection id="schedule-summary-table" resource="scheduleSummary.main">
-        {tableData.data.length === 0 ? (
+        {activities.length === 0 ? (
           <div className="p1 h6 alert">
             {t('scheduleSummary.noDataMessage')}
           </div>
         ) : (
-          <ReactTable
-            showPagination={false}
-            resizable={false}
-            minRows={0}
-            className="h6 -striped"
-            {...tableData}
-          />
+          <table className="table-cms">
+            <tbody>
```

Comment:
It feels a little weird to put headers in a `tbody`, but a table can only have one `thead` and we have header rows spread throughout...  So I guess we can't use `thead`? 😬

Reply:
Hmm. Could we maybe make the `activities.map` function create multiple tables stacked vertically, instead of making it one big table? Then it would be clear which header goes with which rows.

Reply:
Brilliant!